core/node/api_server/src/web3/backend_jsonrpsee/namespaces/eth.rs

```diff
@@ -33,8 +33,13 @@ impl EthNamespaceServer for EthNamespace {
             .map_err(|err| self.current_method().map_err(err))
     }
 
-    async fn estimate_gas(&self, req: CallRequest, block: Option<BlockNumber>) -> RpcResult<U256> {
-        self.estimate_gas_impl(req, block)
+    async fn estimate_gas(
```

Comment:
Any reason new functionality wasn't added to `eth_call`? Geth supports it for `eth_call` AFAICT

Reply:
Also, we have endpoints in zks: estimate_fee and estimate_l1_to_l2_gas, they should be adjusted as well